### ISSUE TITLE
Company Directory

### ISSUE THREAD

**Issue by heldyboy** [ASRG/asrg.io#465]
Do we have the company directory from the old website in the garage?

**Comment by prabath-asrg** [comment]
No, Not yet. I have asked a question in Slack regarding this. let me rephrase it here. 


> On the current ASRG website, there is a directory item called suppliers (https://www.asrg.io/directory/category/suppliers/) and each Item has the following meta fields, Title
Description
Email
Website
Address
Country
Region
City
Zip/Post CodeDo we need these fields like address, region, etc? Since we are now using JetEngine we can have a different set of meta fields if needed. So except for Title, Description, Email, and Website what else we should add as meta fields?

**Comment by heldyboy** [comment]
Company Name
Description
Type = (Supplier, OEM, Consultant, etc.) Multi-Select
Email
Website
Address
Country
Region
City
Zip/Post Code
Logo

Comments, and reviews.

**Comment by heldyboy** [comment]
Still need add a page for this.  Please add this to the menu as directory.

**Comment by prabath-asrg** [comment]
Directory Pages: https://garage.asrg.io/companies/
Directory Single Page: https://garage.asrg.io/companies/google-2/

**Comment by prabath-asrg** [comment]
@prabath-asrg Please make the following changes.
Overview Page
1. Change the title from "Companies" to "Directory"
2. Add "Latest" to the Sort where the newest entries will be sorted first. 
3. I think we want to remove the rating from the overview page.  Only for the Detail / Single Page. 
4. Change Check Profile to Details
5. In Company Type, please change Automotive Security Suppliers to Suppliers
6. In Company Type, please change to Plural... Partner to Partners, etc.
7. Is there a way to Tag specific companies with a label or tag or something?  I want to be able to label companies with specific solutions that they offer.  Like Tools, or VSOC, or SBOM, or Secure Design.
8. Please make the images to fit in the area.
9. 
![image](https://user-images.githubusercontent.com/26025574/189517140-bfbbbde9-cddf-4727-97ea-0101c7a09af2.png)
10. Should be just Load More
![image](https://user-images.githubusercontent.com/26025574/189517166-c03fb412-b2f8-41a8-b47a-32463383b588.png)

**Comment by prabath-asrg** [comment]
Here are the requested changes for the single pages... 
1. Logo should be limited to a size of 200x200 or something much smaller
2. We need to add some fields to the data structure. (Contact person name, Email, Linkedin)
3. Also, can you please add a symbol and link to download the contact as VCard or something like that.  So people can automatically add the contact?

**Comment by prabath-asrg** [comment]
For the archive page, 1-10 is done.

Regarding 7 I can add a taxonomy so we can add tags to the companies.

**Comment by prabath-asrg** [comment]
For Single Pages:

1, 2 are Done.

For 3: Need to search a little to do this. Most probably we may need to add a Custom COde Snippet to do this.

**Comment by usman-asrg** [comment]
After doing the thorough research about this Vcard thing... For every registration into company directory we need to generate one file automatically or manually. and upload that file into either in wordpress or any remote location like google drive, and include that link into the button so user can download or save the information as a contact through that Vcard file.
I was working on this but unable to get it done because of the access issue of siteground. So now its been resolved so I will continue today
